test/bench/pick-layers.bench.js

```diff
@@ -0,0 +1,74 @@
+import {Layer} from 'deck.gl/core';
+import {getClosestFromPickingBuffer} from 'deck.gl/core/lib/pick-layers';
+
+const SAMPLE_LAYERS = [new Layer()];
+const OBJECT_COLOR = [0, 10, 20, 1];
+const NULL_COLOR = [0, 0, 0, 0];
+
+const TEST_CASES = [
+  {
+    title: 'Solid',
+    data: generateSampleData({
+      pickingRadius: 10,
+      getColor: () => OBJECT_COLOR
+    })
+  },
+  {
+    title: 'Circle',
+    data: generateSampleData({
+      pickingRadius: 10,
+      getColor: (x, y) => (10 - x) * (10 - x) + (10 - y) * (10 - y) < 25 ? OBJECT_COLOR : NULL_COLOR
+    })
+  },
+  {
+    title: 'Triangle',
+    data: generateSampleData({
+      pickingRadius: 10,
+      getColor: (x, y) => x + y <= 10 ? OBJECT_COLOR : NULL_COLOR
+    })
+  },
+  {
+    title: 'Solid - Big',
+    data: generateSampleData({
+      pickingRadius: 50,
+      getColor: () => OBJECT_COLOR
+    })
+  }
+];
+
+export default function pickLayersBench(bench) {
+
+  return TEST_CASES.reduce(
```

Comment:
Super-nit: While it may save one line of code, I personally think the choice of `reduce` here compared to a `forEach` or simple `for of` makes the code less readable. But then I never liked reduce in practice...

At the very least replace `b` with `bench` (or maybe `suite` like in the other files).